src/firefly/java/edu/caltech/ipac/firefly/server/catquery/CatMasterTableQuery.java

```diff
@@ -262,7 +262,7 @@ private static String modifyToFullURL(String inStr, String  targetStr) {
                 int endAnchor= s.indexOf(">",start);
                 if (targetStr!=null && endAnchor>0 && url!=null &&
                         !s.substring(beginChar,endAnchor).contains("target=")) {
-                    String rStr= "\\Q" + beginChar+url+beginChar + "\\E";
+                    String rStr = beginChar + url + beginChar;
```

Comment:
Because rStr is used as a regex, simply removing the \Q \E will have a side-effect.  If the url contains '?' for example, the modified code will fail.